src/components/Button/Button.tsx

```diff
@@ -10,8 +10,14 @@ import { ButtonBehavior } from '../../lib/accessibility'
 
 /**
  * A button.
- * @accessibility This is example usage of the accessibility tag.
- * This should be replaced with the actual description after the PR is merged
+ * @accessibility
+ * Default behavior: ButtonBehavior
+ *  - adds role='button' if element type is other than 'button'
+ *
+ *
+ * Other considerations:
+ *  - for disabled buttons, add 'disabled' attribute so that the state is properly recognized by the screen reader
+ *  - if button includes icon only, textual representation needs to be provided by using 'title', 'aria-label', or 'aria-labelledby' attributes
```

Comment:
These are really great points.   They are concise, easy to understand, and easy to implement.  Before merging this, I want to make sure they get first class treatment in the code base.

Here are the thoughts that come to mind when I read the `@accessibility` points above:
1. I want these to stay updated and not get overlooked with library updates.
1. I would like to enforce a test to be written for each point.
1. I would like these points to be accessible from other places.

Perhaps we should move these somewhere closer to the Accessibility classes?  I can see having these in some kind of requirements JSON file where we could then enforce a test to be written for each one.

Alternatively, we could include these as JSDoc comments directly in the Accessibility classes themselves and parse them out like we do props for components.

I'm not sure `Default behavior: ButtonBehavior` makes sense to doc site users without including docs for the accessibility classes, which might be a great idea.

**Conclusion**
This material is too valuable to leave in component comments alone.  We should make it more accessible, put it in the path of updates to keep it fresh, and use it for more cases.

Feedback?

Reply:
I'd recommend having the separate section on the Docs site for accessibility behaviors, the similar as we have Components section. That section should contain all behaviors and description when and how to use them.

Reply:
I agree, especially once the Accessibility classes will include keyboard handling, we should document them separately.

A lot of this information is however related to the component directly, see especially the 'Other considerations' part:
- they include additional information to the user, not something that will be implemented in the behavior.
- they are directly related to the component, less to the behavior
- as they are suggestions, we cannot really test them (it could be however great to come up with a way of automatically notifying user, that once he uses a button with icon only, he should provide a textual representation as well - haven't figured out yet how to do this taking the flexibility of StarDust into account)

I agree that at least big part of the documentation should be inside of the Accessibility objects code - we should generate doc pages based on that and link the component page to its default behavior page.

I would prefer this to be done in a separate PR - this PR already includes valuable information to the users. We can improve both the user facing side of it as well as the way how it is generated, but that should be a separate PR.